src/elements/TextArea/__tests__/TextArea.test.tsx

```diff
@@ -0,0 +1,164 @@
+import { mount, ReactWrapper } from "enzyme"
+import React from "react"
+import { TextArea, TextAreaProps } from "../TextArea"
+
+describe("TextArea", () => {
+  const onChange = jest.fn(({ value }) => {
+    process.stdout.write(value + "\n")
+  })
+  const ref = React.createRef<any>()
+  beforeEach(() => {
+    onChange.mockReset()
+  })
+
+  const simulateTyping = (wrapper: ReactWrapper, text: string) => {
```

Comment:
Nice! Had recently been battling testing keyboard input because of typing, going to see if this can give me a cleaner solution than what I've been using, https://github.com/artsy/positron/blob/a40d397b1fc5237b40b87e3f40b4dcc214cda4f6/src/client/apps/settings/client/curations/display/components/canvas/test/canvas_text.test.tsx#L24